### PR TITLE
fix(filters): datetime for person properties

### PR DESCRIPTION
**Pull request by mariusandra**
## Problem

We're now tracking datetime __person__ properties, but we're not using this information in filters.

## Changes

Make comparisons possible with DateTime fields on persons.

## How did you test this code?

- In the browser 
- Added a tests for propertyDefinitionModel to check it works with non-event properties

### CODE REVIEW

File: frontend/src/lib/api.ts [hunk 1]
```diff
@@ -778,7 +779,7 @@ const api = {
             limit?: number
             offset?: number
             teamId?: TeamType['id']
-            type?: 'event' | 'person' | 'group'
+            type?: string
```
Comment (Twixes) [inline frontend/src/lib/api.ts:782]: What new types are possible outside of `'event' | 'person' | 'group'`?

File: frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx [hunk 2]
```diff
@@ -55,7 +55,9 @@ export function PropertyValue({
 
     const isMultiSelect = operator && isOperatorMulti(operator)
     const isDateTimeProperty = operator && isOperatorDate(operator)
-    const isDurationProperty = propertyKey && describeProperty(propertyKey) === PropertyType.Duration
+    console.log({ type })
```
Comment (Twixes) [inline frontend/src/lib/components/PropertyFilters/components/PropertyValue.tsx:58]: Stray log

File: frontend/src/types.ts [hunk 3]
```diff
@@ -2286,6 +2286,7 @@ export interface PropertyDefinition {
     is_numerical?: boolean // Marked as optional to allow merge of EventDefinition & PropertyDefinition
     is_seen_on_filtered_events?: boolean // Indicates whether this property has been seen for a particular set of events (when `eventNames` query string is sent); calculated at query time, not stored in the db
     property_type?: PropertyType
+    type?: string
```
Comment (Twixes) [inline frontend/src/types.ts:2289]:
```suggestion
    type?: 'event' | 'person' | 'group'
```

File: frontend/src/lib/api.ts (hunk 1 above)
Comment (mariusandra) [reply frontend/src/lib/api.ts]: Sure! Here are 21 more changed files. 😅

**Comment by Twixes** [comment]
Weirdly this doesn't seem to fully work for me. I set the "Browser" property to "Numeric" for persons only (leaving it as "String" for events), yet I still can't see "Larger than" or "Smaller than" when I select the "Browser" person property in a filter – only the string options. Looks like still only the event property definition is used. Can you reproduce this @mariusandra?

**Comment by mariusandra** [comment]
Whoops. Indeed, the `TaxonomicPropertyFilter` discarded the type information when passing the property definitions to `OperatorValueSelect`. Fixed.

**Review by Twixes — APPROVED** [review]

**Pull request merged**